public/form/resetpassword.html

```diff
@@ -1,5 +1,6 @@
 <html lang="en">
 <head>
+	<meta charset="utf-8">
```

Comment:
I agree but this time codification was introduced because copyright symbol wasn't displaying properly. So I guess it's related with the task.